extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/ClientExceptionMapper.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.rest.client.reactive;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.ws.rs.Priorities;
+
+/**
+ * Used to easily define an exception mapper for the specific REST Client on which it's used.
+ *
+ * The annotation MUST be placed on a method of the REST Client interface that meets the following criteria:
```

Comment:
Some of these could be relaxed in the future if users really have good use cases

Reply:
Why static and not a default method?

Reply:
I mean we could add support for default ones later but it's inconsistent with ClientHeaderParam

Reply:
Sure yeah, we could also add support for `default`. I went with `static` because I think it makes more semantic sense.